package.json

```diff
@@ -5,29 +5,31 @@
   "repository": "mitodl/ocw-hugo-themes",
   "main": "base-theme/assets/index.js",
   "scripts": {
-    "lint": "eslint {base-theme,course,course-v2,fields,util,www}",
+    "lint": "eslint {base-theme,course,course-v2,fields,util,www,tests-e2e}",
     "start": "echo \"Run 'npm run start:www' or 'npm run start:course'\"",
-    "start:webpack": "webpack serve --config base-theme/assets/webpack/webpack.dev.ts --hot --host 0.0.0.0 --stats-children",
-    "start:www": "concurrently \"npm:start:webpack\" \"npm:start:www:hugo\" --kill-others",
-    "start:course": "concurrently \"npm:start:webpack\" \"npm:start:course:hugo\" --kill-others",
-    "start:fields": "concurrently \"npm:start:webpack\" \"npm:start:fields:hugo\" --kill-others",
+    "start:webpack": "dotenv -- webpack serve --config ./base-theme/assets/webpack/webpack.dev.ts --hot --host 0.0.0.0 --stats-children",
```

Comment:
This is [dotenv-cli](https://www.npmjs.com/package/dotenv-cli). It loads env vars on the cli.

Reply:
I feel like an idiot now because I didn't realize this package was required for NPM to pass env variables to package scripts. I thought all you needed install was the `dotenv` package and that would happen, but apparently I was mistaken.

Reply:
I actually had never used this package before; I have used
```
 node -r dotenv/config your_script.js
```
(See https://www.npmjs.com/package/dotenv#Preload) but that only works if you're calling node.